Makefile

```diff
@@ -155,11 +158,13 @@ run-docker:
 		-e "NEW_RELIC_LICENSE_KEY" \
 		-e "SECRET_KEY=notasecret" \
 		-p 5000:5000 \
+		--name hypothesis \
 		hypothesis/hypothesis:$(DOCKER_TAG)
```

Comment:
The `--rm` causes us to tidy the container when we quit. The name means we don't create lots of `curious_maxwell` named containers and you can see what it is when it's running.